### ISSUE TITLE
Cannot access JSON data with space in field name

### ISSUE THREAD

**Issue by rmoff** [confluentinc/ksql#2871]
KSQL 5.2.1

Problem: A column in source JSON has spaces in its name. KSQL is not happy about this. 

## Source data 

Send some test data

```
kafkacat -b localhost:9092 -t test01 -P
{ "beat": { "hostname": "xxx", "version": "xxx" }, "Date Time": "2019-05-21T22:18:06.000Z", "host": { "name": "xxx" }, "input": { "type": "xxx" }, "fields": { "env": "xxx" } }
```

Examine in KSQL: 

```
ksql> print 'test01' from beginning;
Format:JSON
{"ROWTIME":1558600742521,"ROWKEY":"null","beat":{"hostname":"xxx","version":"xxx"},"Date Time":"2019-05-21T22:18:06.000Z","host":{"name":"xxx"},"input":{"type":"xxx"},"fields":{"env":"xxx"}}
```

## Create [register] the stream

🔴 Create a stream, using single quotes for the `Date Time` column: 

```
ksql> CREATE STREAM TEST02 (BEAT STRUCT<HOSTNAME VARCHAR, VERSION VARCHAR>, 'Date Time' VARCHAR, HOST STRUCT<NAME VARCHAR>, INPUT STRUCT<TYPE VARCHAR>, FIELDS STRUCT<ENV VARCHAR>) WITH (KAFKA_TOPIC='test01', VALUE_FORMAT='JSON');
line 1:71: extraneous input ''Date Time'' expecting {'INTEGER', 'DATE', 'TIME', 'TIMESTAMP', 'INTERVAL', 'YEAR', 'MONTH', 'DAY', 'HOUR', 'MINUTE', 'SECOND', 'ZONE', 'PARTITION', 'STRUCT', 'EXPLAIN', 'ANALYZE', 'TYPE', 'SHOW', 'TABLES', 'COLUMNS', 'COLUMN', 'PARTITIONS', 'FUNCTIONS', 'FUNCTION', 'ARRAY', 'MAP', 'SET', 'RESET', 'SESSION', 'IF', IDENTIFIER, DIGIT_IDENTIFIER, QUOTED_IDENTIFIER, BACKQUOTED_IDENTIFIER}
Statement: CREATE STREAM TEST02 (BEAT STRUCT<HOSTNAME VARCHAR, VERSION VARCHAR>, 'Date Time' VARCHAR, HOST STRUCT<NAME VARCHAR>, INPUT STRUCT<TYPE VARCHAR>, FIELDS STRUCT<ENV VARCHAR>) WITH (KAFKA_TOPIC='test01', VALUE_FORMAT='JSON');
Caused by: line 1:71: extraneous input ''Date Time'' expecting {'INTEGER',
        'DATE', 'TIME', 'TIMESTAMP', 'INTERVAL', 'YEAR', 'MONTH', 'DAY', 'HOUR',
        'MINUTE', 'SECOND', 'ZONE', 'PARTITION', 'STRUCT', 'EXPLAIN', 'ANALYZE', 'TYPE',
        'SHOW', 'TABLES', 'COLUMNS', 'COLUMN', 'PARTITIONS', 'FUNCTIONS', 'FUNCTION',
        'ARRAY', 'MAP', 'SET', 'RESET', 'SESSION', 'IF', IDENTIFIER, DIGIT_IDENTIFIER,
        QUOTED_IDENTIFIER, BACKQUOTED_IDENTIFIER}
```

✅ Create a stream, using double quotes for the `Date Time` column: 

```
ksql> CREATE STREAM TEST01 (BEAT STRUCT<HOSTNAME VARCHAR, VERSION VARCHAR>, "Date Time" VARCHAR, HOST STRUCT<NAME VARCHAR>, INPUT STRUCT<TYPE VARCHAR>, FIELDS STRUCT<ENV VARCHAR>) WITH (KAFKA_TOPIC='test01', VALUE_FORMAT='JSON');

 Message
----------------
 Stream created
----------------
```

## Query the stream

Describe the schema

```
ksql> describe test01;

Name                 : TEST01
 Field     | Type
-----------------------------------------------------------------------
 ROWTIME   | BIGINT           (system)
 ROWKEY    | VARCHAR(STRING)  (system)
 BEAT      | STRUCT<HOSTNAME VARCHAR(STRING), VERSION VARCHAR(STRING)>
 Date Time | VARCHAR(STRING)
 HOST      | STRUCT<NAME VARCHAR(STRING)>
 INPUT     | STRUCT<TYPE VARCHAR(STRING)>
 FIELDS    | STRUCT<ENV VARCHAR(STRING)>
-----------------------------------------------------------------------
For runtime statistics and query details run: DESCRIBE EXTENDED <Stream,Table>;
```

Query the data

🔴 Nope: 

```
ksql> select * from test01;
Code generation failed for Select: Line 1, Column 17: Unexpected token "Time". expression:TEST01.Date Time, schema:org.apache.kafka.connect.data.SchemaBuilder@13efbc09
Statement: select * from test01;
Caused by: Code generation failed for Select: Line 1, Column 17: Unexpected
        token "Time". expression:TEST01.Date Time,
        schema:org.apache.kafka.connect.data.SchemaBuilder@13efbc09
Caused by: Line 1, Column 17: Unexpected token "Time"
```

🔴 Nope: 

```
ksql> select "Date Time" from test01;
Code generation failed for Select: Line 1, Column 17: Unexpected token "Time". expression:TEST01.Date Time, schema:org.apache.kafka.connect.data.SchemaBuilder@64eb917c
Statement: select "Date Time" from test01;
Caused by: Code generation failed for Select: Line 1, Column 17: Unexpected
        token "Time". expression:TEST01.Date Time,
        schema:org.apache.kafka.connect.data.SchemaBuilder@64eb917c
Caused by: Line 1, Column 17: Unexpected token "Time"
```

----

Related to #2589.
